Lib/Inc/IComChannel.h

```diff
@@ -22,7 +22,7 @@ class IComChannel
 
     virtual bool isCommandAvailable() = 0;
     virtual std::string getCommand() = 0;
-    virtual void send(std::string message) = 0;
+    virtual void send(const std::string &message) = 0;
```

Comment:
Why is it `const` and passed by reference?

Reply:
Because we are not changing the string and it is passed by reference to avoid copying a string.